base_comment_template/views/base_comment_template_view.xml

```diff
@@ -76,6 +76,10 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">base.comment.template</field>
         <field name="view_mode">tree,form</field>
+        <field
+            name="context"
+            eval="{'default_model_ids': [(4, ref('base.model_res_partner'))]}"
```

Comment:
Why applying by default this model? This one is not going to get templates. The target are sales orders, invoices, etc.